content/20.results.md

```diff
@@ -25,34 +25,39 @@ Although Bioinformatics was established in 1998 and BMC Bioinformatics in 2000,
 The overall increasing female proportion trend seems to be present in each honor category, especially in the group of ISCB Fellows (Fig. {@fig:gender_breakdown}C), which seems to have markedly increased after 2015.
 PSB has only one or two keynotes per year, so its estimated proportion of female speakers varies considerably (Fig. {@fig:gender_breakdown}C).
 However, the 2020 PSB keynotes were the only set of honorees across our analysis for which nearly all the probability was ascribed to female speakers.
 
 
-![Estimated proportion of gender prediction over the years of all Pubmed journal authors (A),
-  of all ISCB fellows and keynote speakers (B),
-  and of ISCB honorees in each honor category (C).
+![Estimated composition of gender prediction over the years of 
+  (A) all Pubmed computational biology and bioinformatics journal authors,
+  (B) all ISCB fellows and keynote speakers,
+  and (C) ISCB honorees in each honor category.
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/master/figs/gender_breakdown.png){#fig:gender_breakdown}
 
 ### Assessing the Racial and Ethnic Diversity of Authors and Honorees
 
 We predicted the race and ethnicity of authors and honorees using wru, which is based on US census data.
-We found that an increasing proportion of authors in computational biology and bioinformatics journals had last names associated with selecting Asian as a race/ethnicity category in the US census (Fig {@fig:racial_makeup}A).
-This was primarily driven by publications in _Bioinformatics_ and _BMC Bioinformatics_ (Fig {@fig:racial_makeup}B, top).
-We did not observe an increase at PLOS Computational Biology (Fig {@fig:racial_makeup}B, bottom).
-A higher proportion of individuals who had last names associated with selecting white as a race/ethnicity category in the US census were ISCB honorees (Fig {@fig:racial_makeup}C) than authors (Fig {@fig:racial_makeup}A).
-Separating honoree results by honor category did not reveal any clear differences (Fig {@fig:racial_makeup}D).
-
-![Estimated proportion of census-based race prediction over the years of all Pubmed journal authors (A),
-  of authors in each computational biology and bioinformatics journal (B), of all ISCB fellows and keynote speakers (C),
-  and of ISCB honorees in each honor category (D).
+We found that an increasing proportion of authors in computational biology and bioinformatics journals had last names associated with selecting Asian as a race/ethnicity category in the US census (Fig. {@fig:racial_makeup}A).
+This was primarily driven by publications in _Bioinformatics_ and _BMC Bioinformatics_ (Fig. {@fig:racial_makeup}B, top).
+We did not observe an increase at PLOS Computational Biology (Fig. {@fig:racial_makeup}B, bottom).
```

Comment:
```suggestion
We did not observe an increase at _PLOS Computational Biology_ (Fig. {@fig:racial_makeup}B, bottom).
```
I think that at the end we're just going to have to search the journal names to make sure we got them underlined. 😆